recirq/qml_lfe/circuit_blocks.py

```diff
@@ -0,0 +1,162 @@
+# Copyright 2020 Google
```

Comment:
I know it doesn't feel like it sometimes, but it is actually 2021.

Reply:
Fixed.